include/votca/tools/edge.h

```diff
@@ -36,29 +37,31 @@ namespace tools {
  * a link.
  */
 class Edge {
- private:
-  std::pair<int, int> vertices;
+ protected:
+  std::vector<int> vertices_;
 
  public:
   Edge() {}
   ~Edge() {}
   /// Creates an edge the smallest integer value will be placed in the id1
   /// spot and the larger in the id2 spot
   Edge(int ID1, int ID2);
-  Edge(const Edge& ed) : vertices(ed.vertices) {}
+  Edge(const Edge& ed) : vertices_(ed.vertices_) {}
```

Comment:
the compiler creates that copy constructor so just delete this please.

Reply:
Ok